Release Mumble 1.5.629 (RC 3) (#208)
mumble-voip/mumble-www: hugo/content/blog/2024-04-14-mumble-1-5-629.md

---
title: Mumble 1.5.629 (RC 3)
author: Hartmnt
date: 2024-04-14
categories:
  - Release
  - Release Candidate
---

We are happy to release the third and hopefully final release candidate of the new stable series Mumble 1.5.x. It fixes some issues and finally introduces the long-awaited accessibility overhaul of the entire application.

Also, as requested by popular demand, we reintroduce RNNoise in the form of our own [fork](https://github.com/mumble-voip/ReNameNoise). This should hopefully prevent any future issues while keeping a beloved Mumble feature.

> **Note**: If want to use RNNoise and you are upgrading from the previous RC (1.5.613), you will need to manually re-enable RNNoise in the settings.

Reviewer: ```suggestion
**Note**: If you want to use RNNoise and you are upgrading from the previous RC (1.5.613), you will need to manually re-enable RNNoise in the settings.
```

You can download the new version from our [**Downloads page**]({{< relref "/downloads#development-snapshots" >}}) (under snapshots) or the
[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.5.629).

<!--more-->

## Changelog

> This is the changelog for this release (changes relative to the previous RC2 Mumble 1.5.613).

Reviewer: Would be nice if we linked the previous RC release notes.

> This is the changelog for this release (changes relative to the previous RC2 Mumble 1.5.613).

Reviewer: suggestion

```suggestion
This is the changelog for this RC release since the previous RC release RC2 Mumble 1.5.613.
```
